ui/tooltip.js

```diff
@@ -25,9 +25,12 @@
 	}
 }(function( $ ) {
 
-return $.widget( "ui.tooltip", {
+$.widget( "ui.tooltip", {
 	version: "@VERSION",
 	options: {
+		classes: {
+			"ui-tooltip": "ui-corner-all"
```

Comment:
This needs to include "ui-widget-shadow" once the theme update is in master. I hope we can finish that one today or tomorrow, to then integrate it here.